File: Leetcode/JS/Leetcode2029-石子游戏IX.js
var stoneGameIX = function(stones) {
    const cnt = [0,0,0];
    for(const st of stones) cnt[st % 3]++;
    return cnt[0] % 2 == 0 ? cnt[1] >= 1 && cnt[2] >= 1 : Math.abs(cnt[1] - cnt[2]) > 2;
};

File: Leetcode/PY/Leetcode2029-石子游戏IX.py
class Solution:
    def stoneGameIX(self, stones: List[int]) -> bool:
        cnt = [0] * 3
        for st in stones: cnt[st % 3] += 1
        if cnt[0] % 2 == 0: return cnt[1] >= 1 and cnt[2] >= 1
        return abs(cnt[1] - cnt[2]) > 2